Makefile

```
@@ -590,6 +590,7 @@ install-test-tools-local:
 .PHONY: install-codegen-tools-local
 install-codegen-tools-local:
 	./hack/install.sh codegen-tools
+	./hack/install.sh goreman
```

```suggestion
	./hack/install.sh goreman
```

I don't think that you're required to install goreman as a part of codegen-tools. Please verify this again.

---

@nitishfy , previously I added it by mistake in `install-test-tools-local`, coz it was for installation of tools for e2e tests.
And, there are only 3 tools to be installed as mentioned [here](https://github.com/yaten2302/argo-cd/blob/ea2585c2159de089f199be118957be07ec209532/Makefile#L581). Just wanted to confirm, is this to be added in installing the Go tools? - `install-go-tools-local`?